README.rst

```diff
@@ -20,7 +20,7 @@ Dependencies
 ------------
 
 * Python 2.6+ (not Python 3)
```

Comment:
No, I just said that because we may have been testing against 2.6 in the past and though the README said we support 2.6, if we aren't testing against it, I don't think we do support it now.